src/functions-and-arrays.js

```diff
@@ -52,14 +124,44 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  let newArr = []
+  if (wordsUnique.length === 0) {
+    return null
+  }
+  for (let i = 0; i < wordsUnique.length; i++) {
+    if (wordsUnique.indexOf(wordsUnique[i], i + 1) == -1) {
```

Comment:
You may want to check this logic again. You have to check the indexOf from the newArr